app/javascript/components/inputs.js

```diff
@@ -14,81 +14,51 @@ export function SingleInput ({ id, label, placeholder, fullWidth, type = "text"
       placeholder= {placeholder}
       component='input'
       type={type}
+      min={type == 'number' ? 0 : null}
+      readOnly={!isEditable}
```

Comment:
just a general observation; you're using `isEditable` in every component to set the state of `readOnly`. In every occasion you have to negate it, so why don't you use `readOnly` instead?

Reply:
yeah I didn't like it, was jamming it in at the end of the day, have changed it now